template/index.html

```diff
@@ -241,21 +241,20 @@ <h3 class="UseCase-title">DevOps &amp; Site Reliability</h3>
         <div class="LearnGo-header">
             <h2>Learn Go</h2>
             <p>
-                “I started writing in Go when I was just out of college and couldn’t believe how quickly I picked it up,
-                especially compared to other languages. I had spent months learning about how to build APIs in other languages
-                (and mostly failing), but <b>after a week of writing Go, I had completed my first API</b>.”
+                “当我刚从大学毕业时就开始用Go编程，相比其他语言Go的上手速度之快让我难以相信。
+                我曾花费数月时间学习如何利用其他语言创建API(多数失败了)，但<b>当使用Go编程仅一周后，我就完成了自己的第一个API</b>”
             </p>
             <p style="text-align: right;">
                 -
                 <a href="https://medium.com/capital-one-tech/good-to-go-a-language-that-makes-it-easy-for-engineers-to-talk-to-each-other-17ad0f47f76c"
                    target="_blank">
-                    Kaylyn Gibilterra, Senior Software Engineer, Capital One
+                    Kaylyn Gibilterra, 高级软件工程师, 美国第一资本金融公司
```

Comment:
```suggestion
                    来自 Capital One 的高级软件工程师，Kaylyn Gibilterra
```

公司名不用翻译感觉，除非微软、谷歌这种国内译名知名度比较高的。。其实我觉得之后应该全部换成在国内任职的工程师语录，哈哈